std_msgs/Header header
geometry_msgs/PoseStamped[] poses
recycling_stretch/SimDetectedBox sim_detected_box_data
